### ISSUE TITLE
Exo 30:33 UDB  Who cuts a person off?

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/udb-en#96]
ULB "that person must be cut off from"
We have a note on 30:33 saying to see how they translated this in 12:15. But the UDB interprets that phrase differently in 12:15 and 30:33.  In 12:15 UDB, it is the people who should cut off the person, while in  30:33 UDB, it is God who considers the person to be cut off. Should the UDB translate it the same way in both verses, or should I get rid of the note?

Exo 12:15 ULB  that person must be cut off from Israel. 
Exo 12:15 UDB **you** must drive that person out from your people. 

Exo 30:33 ULB that person must be cut off from his people.
Exo 30L:33 UDB **I will view** [that person] as no longer belonging to my people."

**Comment by SusanQuigley** [comment]
Also, If God is cutting the person off, then "his people" could be either "God's people" or "the person's people."
If the people are cutting the person off, then it would seem that  the word "his" would have to refer to the people.

**Comment by SusanQuigley** [comment]
This issue also comes up in 30:38.

**Comment by hmw3** [comment]
This issue comes up in 
Genesis 17:12-14
Exodus 30:32-33
Exodus 30:37-38
Exodus 31:12-15
Leviticus 07:19-20
Leviticus 07:21
Leviticus 07:25-27
Leviticus 17:1-4
Leviticus 17:8-9
Leviticus 18:29-30
Leviticus 19:5-8
Leviticus 20:17-18
Leviticus 23:28-29
Numbers 09:13-14
Numbers 15:30-31
I think it needs a consistent policy.

**Comment by TomWarren** [comment]
Recommendations for consistent policy?

**Comment by pohlig** [comment]
The problem is that the expression is ambiguous, as is exactly reflected by the variation of treatment in the UDB. Be cut off can mean, acc. to the commentators et al., be killed, be banished or driven off from the community of Israel--either by the people themselves or by God.
To make matters worse, "cut off" in other contexts sometimes unambiguously means "destroy." In those cases, "Be cut off from..." exactly parallels "be destroyed from," as in, "be destroyed from the face of the earth."
What's the solution? I doubt that a totally consistent policy is the right answer, because contexts vary. And however much consistency you impose, someone else will always disagree with your choice. The best solution I can think of is to try for as much consistency as possible and then to use a note to see that some versions interpret your choice differently. I'll do it if you want. Tell me what you want.

**Comment by hmw3** [comment]
Since my feeling is that the more chunks are affected by a decision the higher up the totem pole the decision needs to be made, and since you're the most likely to get it done to your satisfaction, I'd like you to do it.

**Comment by SusanQuigley** [comment]
I could get rid of the notes saying "See how you translated this in 12:15" and allow each chunk to stand on its own. 

How would this be for a note? I could reorder the possible meanings, putting the meaning that fits with the UDB first.

**that person must be cut off from his people** - Possible meanings are 1) God is telling the Israelites to make that person leave their community, or 2) God is telling the Israelites to kill that person, or 3) God will not consider that person to be one of his people.

**Comment by pohlig** [comment]
that sounds pretty good, Susan, and yr proposal seems to amount more or less to what i was proposing myself. i also like your proposed note. 
i am still okay with me myself going thru the whole list and applying yr nice note, reordering the possible meanings as appropriate to each passage. or you can do it. either way.

**Comment by TomWarren** [comment]
Thanks, Jim.

**Comment by SusanQuigley** [comment]
If you think you might want to change the UDB for any of these, it would be good for you to write the notes so we know that the notes fit with the UDB. If we assume that the UDB is fine, then I can do the notes.

**Comment by SusanQuigley** [comment]
No need to change Exodus UDBs unless at some point we want to make sure that they are the same as truly parallel passages in other OT books.

I used variations of this note in the five "be cut off from" passages in Exodus, putting the UDB interpretation first.
**must be cut off from his people** - The metaphor "cut off" has at least three possible meanings. They can be expressed in active form: 1) "I will no longer consider him to be one of the people of Israel" 2) "the people of Israel must send him away" or 3) "the people of Israel must kill him." (See: Metaphor and Active or Passive)

Exo 12:15 and 19 deal with the person who eats yeast at  Passover
12:15 you must drive that person out from your people.
12:19  that person will no longer be an Israelite.  (I added a note to see 12:14.)

Exodus 30:33 and 38 deal with the person who uses God's incense.
30:33  will view [him], as no longer belonging to my people.
30:38  I will view [him] as no longer belonging to my people. (I added a note to see 30:33.)

Exo 31:14 deals with the person who works on the Sabbath.
31:14 this will show that I no longer consider them to belong to my people.
